### DISCUSSION TITLE
Setting to disable player morphs

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#3866] [Suggestions]
Much like transmog, player morphs are a cool feature, but not one that everyone likes

There should be a command to disable them for your own perspective

**Comment by Vitusin** [comment]


**Reply by BeefSteaker** [reply]
I'm talking about this https://www.chromiecraft.com/en/costumes-player-morphs-now-available-on-chromiecraft/

Not xmog

**Comment by ishidinho** [comment]
Afaik, there is such a command. Maybe ask in Discord.

**Reply by BeefSteaker** [reply]
Not Currently 
![Screenshot_20220818-140745](https://user-images.githubusercontent.com/11332559/185402620-a24a7daf-9c9b-4c41-88b5-6a9db83399b0.png)
